### PR TITLE
Update Try Dart homepage pad to use null safety

### PR DESCRIPTION
**Pull request by parlough**
Update the "Try Dart" DartPad on the home page to support null safety and update examples appropriately.

cc @johnpryan

### CODE REVIEW

**Review by kwalrath — COMMENTED** [review]
Nice work, @parlough!

File: tool/dartpad_picker/web/dartpad_picker_main.dart [hunk 1]
```diff
@@ -52,13 +52,13 @@ bool isMobile() {
...
-var helloWorld = r'''
-main() {
+const helloWorld = r'''
+void main() {
   print("Hello, World!");
 }
-'''
-    .trim();
-var functions = r'''
+''';
```
Comment (johnpryan) [inline tool/dartpad_picker/web/dartpad_picker_main.dart:59]: Is trimming not necessary?

Comment (parlough) [reply tool/dartpad_picker/web/dartpad_picker_main.dart]: It changed the extra behavior of the line at the end, but in my latest commit I've moved the multi-line literal right after the previous bracket to match the earlier behavior so the strings are equivalent while still allowing them to be const.

Comment (johnpryan) [reply tool/dartpad_picker/web/dartpad_picker_main.dart]: cool, LGTM

**Review by johnpryan — APPROVED** [review]

**Pull request merged**